### ISSUE TITLE
when Post-trained  douban  dataset  the loss function shake little,is that correct?

### ISSUE THREAD

**Issue by andrew-begain** [hanjanghoon/BERT_FP#4]
dear hanjanghoon
thanks for you greate work! when i Post-trained  douban  dataset  the loss function shake around 2.0 ,is that correct?

**Comment by hanjanghoon** [comment]
I remember that the loss during post-training was about 1.7~2.0, and about 0.2 during finetuning.

**Comment by andrew-begain** [comment]
> I remember that the loss during post-training was about 1.7~2.0, and about 0.2 during finetuning.

@hanjanghoon thanks for your reply, How does this algorithm apply to a real conversation system?  It's a question and a answer format .thank you very much
